### ISSUE TITLE
Can't open link to shared_ptr/operator*

### ISSUE THREAD

**Issue by Zitrax** [zealdocs/zeal#1489]
For the C++ documentation the link to `operator*` and `operator->` gives QtNetwork Error 203.

Using Zeal 0.6.1 on Windows 10.

Full error output:

```
Error opening C:/Users/MyName/AppData/Local/Zeal/Zeal/docsets/C++.docset/Contents/Resources/Documents/en.cppreference.com/w/cpp/memory/shared_ptr/operator*.html: Felaktig syntax för filnamn, katalognamn eller volymetikett

Failed to load URL file:///C:/Users/MyName/AppData/Local/Zeal/Zeal/docsets/C++.docset/Contents/Resources/Documents/en.cppreference.com/w/cpp/memory/shared_ptr/operator*.html.

QtNetwork Error 203
```

**Comment by trollixx** [comment]
Duplicate of #1278. Unfortunately, not an easy fix and depends on #138.
